### PR TITLE
[wasm] Perf pipeline - use `stable` version of `v8`

### PR DESCRIPTION
**Pull request by radical**
- Use V8 version from ChromeVersions.props
  - This affects only the perf pipelines
  - V8 with the given version is installed, and then used on helix
  - The version will be updated with the chrome browser versions, in
    `eng/testing/ChromeVersions.props`.

**Note**: The perf pipelines were running with `v8` from the `dev` channel
earlier, but with this change `v8` from the `stable` channel will be
used instead.

- Emit V8 version also in ChromeVersions.props

### CODE REVIEW

**Comment by radical** [comment]
Test run: https://dev.azure.com/dnceng/internal/_build/results?buildId=2250017&view=results .

File: eng/pipelines/coreclr/templates/run-performance-job.yml [hunk 1]
```diff
@@ -64,12 +64,14 @@ jobs:
     - ${{ if eq(parameters.runtimeType, 'wasm') }}:
       - HelixPreCommandsWasmOnLinux: >-
           sudo apt-get -y remove nodejs &&
-          curl -fsSL https://deb.nodesource.com/setup_16.x | sudo -E bash - &&
+          curl -fsSL https://deb.nodesource.com/setup_18.x | sudo -E bash - &&
```
Comment (kg) [inline eng/pipelines/coreclr/templates/run-performance-job.yml:67]: We should consider making a note that this needs to be updated every time we bump our linux version

**Review by kg — APPROVED** [review]

File: eng/pipelines/coreclr/templates/run-performance-job.yml (hunk 1 above)
Comment (radical) [reply eng/pipelines/coreclr/templates/run-performance-job.yml:67]: I'll add this in a follow up PR.

**Comment by lewing** [comment]
This looks like it probably caused a lot of perf changes so be aware.

**Pull request merged**